0.020198836540065934: catch( AssertionError e ) { error = e ;
0.020198836540065934: catch( AssertionError e ) { error = error ;
0.023778960072570188: catch( AssertionError e ) {
0.036636010929100404: catch( AssertionError e ) { error =
0.044210360703535345: catch( AssertionError e ) { error = null ;
0.05688087485698479: catch( final AssertionError e ) {
